kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -56,6 +58,16 @@ func SignedBlindedTokenIssuerHandler(
 			})
 			continue
 		}
+		// validate that the number of blinded tokens is evenly divisable
+		if len(request.Blinded_tokens)%issuer.Buffer != 0 {
+			blindedTokenResults = append(blindedTokenResults, avroSchema.SigningResult{
+				Signed_tokens:     nil,
```

Comment:
should this be camelCase?

Reply:
avro generated type doesn't follow our linting style unfortunately